gcp_variant_transforms/options/variant_transform_options.py

```diff
@@ -124,9 +124,27 @@ class BigQueryWriteOptions(VariantTransformsOptions):
 
   def add_arguments(self, parser):
     # type: (argparse.ArgumentParser) -> None
-    parser.add_argument('--output_table',
-                        default='',
-                        help='BigQuery table to store the results.')
+    parser.add_argument(
+        '--output_table',
+        default='',
+        help=('Base name of the BigQuery tables which will store the results. '
+              'Note that sharded tables will be named as following: '
+              ' * `output_table`__chr1 '
+              ' * `output_table`__chr2 '
+              ' * ... '
+              ' * `output_table`__residual '
+              'where "chr1", "chr2", ..., and "residual" suffixes correspond '
+              'to the value of `table_name_suffix` in the sharding config file '
+              '(see --sharding_config_path).'))
+
+    parser.add_argument(
+        '--sample_lookup_optimized_output_table',
+        default='',
+        help=('In addition to the default output tables (which are optimized '
+              'for variant look up queries), you can store a second copy of '
+              'your data in BigQuery tables that are optimized for sample '
+              'look up queries. Note that setting this option will double your '
```

Comment:
Since the tables are going to be unflattened, are their storage costs going to be more than the original output_tables? Or do they end up roughly equal?

Reply:
If data is not joint genotyped then it's approximately equal.
I updated the help to make this issue clear, thanks for spotting it.